src/alert.html

```diff
@@ -21,6 +21,12 @@
   {{ pageTitle }}
 {%- endblock %}
 
+{% if alert_data.is_expired %}
```

Comment:
Can we also set the icon to grey on this page is the alert is expired?

Reply:
Sure, I've amended the last commit to do that 👍.